### PR TITLE
change image

### PR DESCRIPTION
**Pull request by hubi0111**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "ieee-ghost",
+  "version": "1.0.0",
+  "lockfileVersion": 1
+}
```
Comment (PotatoParser) [inline package-lock.json:5]: Looks like you accidentally ran `npm install` within the main directory. Please remove this file.

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Looks good! Just one file that needs to be removed.

File: package-lock.json (hunk 1 above)
Comment (hubi0111) [reply package-lock.json:5]: hmm, I recommitted but the commit isn't showing up. should I just open a new request?

Comment (PotatoParser) [reply package-lock.json:5]: Did you push the commit?

Comment (PotatoParser) [reply package-lock.json:5]: It looks like you commit on the same branch that I already merged into master, please make commits on a new branch that's based off master (not a stale branch).

**Comment by PotatoParser** [comment]
Closing this since it should be a new branch

**Pull request closed without merging**